Tests/DatadogTests/Datadog/Core/SessionTracerTests.swift

```diff
@@ -0,0 +1,106 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+import Datadog
+import _Datadog_Private
+
+private extension String {
+    init(fromUnsafePtr unsafePtr: UnsafePointer<Int8>) {
+        self.init(format: "%s", unsafePtr)
+    }
+}
+
+class SessionTracerTests: XCTestCase {
```

Comment:
I think we rather have `Swizzler` tests in there as all of them assert things with reflection. We also need the instrumentation unit tests which use `DDTracer()` and `SpanOutputMock()` for recording and asserting spans:
```swift
func testGivenInstrumentedURLSession_whenResponseIsReceived_itSendsSpan() {
   // test 200 OK response
   // test error response
}
```

Reply:
Moved them to `SwizzlerTests` ✅